lib/migration/config/v1/executor/container_vm.go

```diff
@@ -0,0 +1,218 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
At some point, we'll need to change the year in the [copyright checker script](https://github.com/vmware/vic/blob/master/infra/scripts/VMware-copyright#L1) and update all files in one go. @mdubya66?